src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/ReadyToRunCompilerContext.cs

```diff
@@ -67,6 +69,8 @@ public ReadyToRunCompilerContext(TargetDetails details, SharedGenericsMode gener
             }
         }
 
+        public InstructionSetSupport InstructionSetSupport { get { return _instructionSetSupport; } }
```

Comment:
nit: Auto-implemented get-only property instead of manual read-only field + property?